AI/DB_interface.py

```
@@ -0,0 +1,54 @@
+def close(conn): # closes the connection
+    if conn: # if the connection exists
+        conn.close() # close it
+
+def get_data(query, params=()): 
+    conn = connect() # connects to the DB
+    try:
+        cursor = conn.cursor() # creates a cursor object
+        cursor.execute(query, params) # executes the query with params
+        rows = cursor.fetchall() # fetches all the data
+        close(conn) # closes the connection
+        return rows # returns the data
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)